generators/app/tasks/appSetup/fixBundleIdentifier.js

```diff
@@ -5,7 +5,7 @@ module.exports = function fixBundleIndentifier() {
   const replaceRegex = new RegExp(`PRODUCT_NAME = ${this.projectName};`, 'g');
   const fixedProjectContent = iosProjectContent.replace(
     replaceRegex,
-    `PRODUCT_BUNDLE_IDENTIFIER = ${this.bundleId};\n\t\t\t\tPRODUCT_NAME = ${this.projectName};`
+    `PRODUCT_BUNDLE_IDENTIFIER = ${this.bundleId};\n\t\t\t\tPRODUCT_NAME = ${this.projectName};\n\t\t\t\tTARGETED_DEVICE_FAMILY = "1,2";`
```

Comment:
@sbalay Is it correct to add `TARGETED_DEVICE_FAMILY` here ? I think this is nothing to do with the Bundle Identifier, but I found convenient to append this field here.